src/winforms/toga_winforms/window.py

```diff
@@ -111,6 +112,11 @@ def show(self):
         if self.interface is not self.interface.app._main_window:
             self.native.Show()
 
+    def on_formclose(self, event, handler):
+        if self.interface.app.on_exit:
+            self.interface.app.on_exit(self.interface.app)
+        print('On form close')
```

Comment:
This looks like some leftover debugging code? I'm actually in favor of having toga event logging in some central place, but I'm not sure print statements are what we want.

Reply:
It is debugging, and I ran a search for print statements before committing, but I guess I missed a file. Sorry about that.